src/contentful-typescript-codegen.ts

```diff
@@ -29,31 +29,31 @@ const cli = meow(
       output: {
         type: "string",
         alias: "o",
-        required: true,
+        isRequired: true,
```

Comment:
`isRequired` is the correct name for this key. See https://github.com/sindresorhus/meow#flags